### ISSUE TITLE
Updating product images does not change updated_at field in REST API or database

### ISSUE THREAD

**Issue by fvschie** [magento/magento2#8280]
### Preconditions

1. Any Magento2 version.
2. Functional REST API.

### Steps to reproduce

1. Retrieve product via REST API, eg.: GET http://magento.dev:9080/rest/V1/products/MT07
2. Check updated_at field.
3. Delete, add or change an image (even just changing the base image)
4. Retrieve the same product via REST API, eg.: GET http://magento.dev:9080/rest/V1/products/MT07

### Expected result

1. Database (and REST API) field updated_at should be set to the date of the change, as the product changed.

### Actual result

1. The change date is not stored with the product; an external API has no way to detect the image is different other than storing image urls and comparing them.

**Comment by fvschie** [comment]
Any updates?

**Comment by amnotafraid** [comment]
I am using Magento 2.1.3.  If I get a product (replace base_url with correct magento 2 url; replace token with correct token; and use a valid sku):
```
curl -X GET "base_url/rest/V1/products/IHAR00000002" -H "Authorization: Bearer token" -H "Content-Type:application/json"
```
I get this (please note name and updated_at fields):
```
{
  "id": 3,
  "sku": "IHAR00000002",
  "name": "ER 308 / 308L Stainless MIG Wire .035 X 25# Spool",
  "attribute_set_id": 10,
  "price": 265,
  "status": 1,
  "visibility": 4,
  "type_id": "simple",
  "created_at": "2013-05-13 19:50:31",
  "updated_at": "2016-09-23 23:04:55",
...
```
If I then change the name to George (replace base_url with correct magento 2 url; replace token with correct token; and use the same  sku as before):
```
curl -X PUT "base_url/rest/V1/products/IHAR00000002" -H "Authorization: Bearer token" -H "Content-Type:application/json" -d '{"product": {"name": "George"}}'
```
And then I repeat the GET, I get the following.  Please note that the name is correctly changed to George, but the updated_at is incorrect.  It should have changed with the PUT:
```
{
  "id": 3,
  "sku": "IHAR00000002",
  "name": "George",
  "attribute_set_id": 10,
  "price": 265,
  "status": 1,
  "visibility": 4,
  "type_id": "simple",
  "created_at": "2013-05-13 19:50:31",
  "updated_at": "2016-09-23 23:04:55",
...
```

**Comment by RomanKrut** [comment]
@fvschie @amnotafraid  Thanks for Your report and comment. Internal tickets were created MAGETWO-70179 and MAGETWO-70180.

**Comment by careys7** [comment]
Related: #6683

**Comment by magento-engcom-team** [comment]
@fvschie, thank you for your report.
The issue is already fixed in develop branch, 2.2.0
